tests/models/test_gpu.py

```diff
@@ -93,61 +95,37 @@ def device_count():
 
 
 @pytest.mark.parametrize(
-    ["gpus", "expected_num_gpus", "strategy"],
+    ["devices", "expected_num_gpus", "strategy"],
     [
-        pytest.param(None, 0, None, id="None - expect 0 gpu to use."),
-        pytest.param(0, 0, None, id="Oth gpu, expect 1 gpu to use."),
```

Comment:
could you remind me why we are removing these?

Reply:
Because `gpus=0` or `gpus=None` means the accelerator=cpu. The file is, however, `test_gpu.py`. Also, setting devices=0 or devices=None gives an error.